app/models/stage.rb

```diff
@@ -211,6 +211,10 @@ def locked_by?(lock)
     end
   end
 
+  def as_json(options = {})
+    super(options).merge(deploy_groups: deploy_groups.as_json)
```

Comment:
don't include things by default into every stage reply ... manually add it in the webhook or something like that